src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,35 @@
+@use "../utils/_variables" as *;
+@use "../utils/_mixins" as *;
+@use "../utils/_functions" as *;
+@use "../blocks/reset.scss" as *;
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: #{$cell-gap};
+  width: calc(7 * (#{$cell-size}) + 10 * #{$cell-gap});
+
+  &__cell {
+    @include square ($cell-size);
+    background-color: #{$cell-background-color};
+    border: #{$cell-border};
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
```

Comment:
Please, organize your styles 
I recommend you take a look at these styles code and use it in the feature
[9elements](https://9elements.com/css-rule-order) or [htmlAcademy](https://codeguide.academy/html-css.html#css-order)